src/main/scala/ai/privado/exporter/JSONExporter.scala

```diff
@@ -223,6 +222,30 @@ object JSONExporter {
     }
   }
 
+  def dedSourceDiscoveryAuditFileExport(
+    outputFileName: String,
+    repoPath: String,
+    dedSourceData: List[DEDSourceAudit]
+  ): Either[String, Unit] = {
+    logger.info("Initiated the Intermediate exporter engine")
+    val output = dedSourceData.asJson
+    try {
+      logger.info("Completed Intermediate DED Source Discovery Exporting")
+
+      val f = File(s"$repoPath/$outputDirectoryName/$outputFileName")
+      f.write(output.asJson.toString())
+      logger.info("Shutting down Intermediate Exporter engine")
+      Right(())
+
+    } catch {
+      case ex: Exception =>
+        println("Failed to export intermediate output")
```

Comment:
We don't need to print the audit, intermediate output, as the end user doesn't care about these, we can have this in log files